src/baseevents.cpp

```diff
@@ -74,6 +74,7 @@ bool BaseEvents::loadFromXml()
 
 		if (!success || !registerEvent(event, node)) {
 			delete event;
+			return false;
```

Comment:
Yes but it dosn't seem to break anything, it would always return true before this change.